docs/src/@primer/gatsby-theme-doctocat/nav.yml

```diff
@@ -82,6 +82,8 @@
       url: /Heading
     - title: IconButton
       url: /IconButton
+    - title: KeybindingHint
+      url: /KeybindingHint
```

Comment:
Should this be under drafts as well in the nav

(non blocking because these docs are going away soon anyways in favor of https://primer.style/components)